FacebookEventForwarder.js

```diff
@@ -172,6 +177,23 @@
                         }, 0);
                         params['num_items'] = num_items;
                     }
+                    else if (event.ProductAction.ProductActionType == mParticle.ProductActionType.RemoveFromCart) {
+                        eventName = 'RemoveFromCart';
+
+                        totalValue = event.ProductAction.ProductList.reduce(function(sum, product){

```

Comment:
I'm curious why perform the sum here, and not just set the TotalAmount. Esp since below you're checking for TotalAmount and then setting the sum if TotalAmount exists. My feeling is it should work the same for all ProductActionType's, and just use event.ProductAction.TotalAmount